### PR TITLE
Adding offline installer 

### PR DESCRIPTION
**Pull request by shailendrarjpt**
Copied the Nephio-ansible-install directory to Nephio-ansible-install-offline directory to add offline installer.
Added clab install offline.

### CODE REVIEW

**Comment by ganchandrasekaran** [comment]
fix  https://github.com/nephio-project/one-summit-22-workshop/issues/145

**Comment by henderiw** [comment]
Why are we not making this an option of the existing install? User could have the option to choose one or the other through an option.

**Comment by shailendrarjpt** [comment]
> 

Sure, is it okay to have a property like "installation_mode: offline/online" and use this property in the yaml task files to execute online or offline installation?

File: nephio-ansible-install/README.md [hunk 1]
```diff
@@ -56,6 +57,8 @@ all:
     host_min_root_disk_space: 50 # minimum required disk space before install; value in GB
     tmp_directory: "/tmp"
     bin_directory: "/usr/local/bin"
+    installer_lib: <provide full path for installer_lib direcotry under nephio-ansible-install-offline>
```
Comment (vjayaramrh) [inline nephio-ansible-install/README.md:60]: what would the value for this field look like? Is nephio-ansible-install-offline a directory?

Comment (shailendrarjpt) [reply nephio-ansible-install/README.md:60]: This is the directory to keep library files for offline installation.

Comment (shailendrarjpt) [reply nephio-ansible-install/README.md:60]: For using the offline installation, we just need to set installation_mode property in nephio.yaml to offline.

**Comment by johnbelamaric** [comment]
Looks like it needs a rebase.

**Comment by shailendrarjpt** [comment]
> [ ](/shailendrarjpt)

Hey I resolved the conflicts and rebased but it still showing conflicts.
Now if I am doing rebase in VScode there is no conflicts its showing.

Please let me know how to rebase it to fix this.

**Comment by johnbelamaric** [comment]
I used the GH UI to try to resolve the conflicts...let's see how it looks after that.

**Comment by johnbelamaric** [comment]
Please review the change I made...

**Comment by shailendrarjpt** [comment]
> Please review the change I made...

Looks good to me, thanks!

**Pull request merged**